lib/engine/ability/unparrable.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
```

Comment:
why is this needed, can't you just define can_par? in game like others?

Reply:
I did some changes to remove this ability; I also defined `float_str` which will communicate the same information to the player
![image](https://user-images.githubusercontent.com/2993555/103465905-f5484580-4d0d-11eb-997e-60d2474d5137.png)